README.md

```diff
@@ -60,3 +60,13 @@ git add (files)
 git commit -m "message"
 git push origin (branchname)
 ```
+
+## Update the Maps
+
+To update the maps, you will need to determine which layer needs updating. If it is a ward, public building, etc, then you need to update the geojson files in the Rails application. If it is a directory layer, like the breakdowns of court roles, gender, occupation, and race, then you should modify the directory CSV file in `source/csv`. Once your CSV is updated, you may run:
+
+```
+post -f csv -x html -e [environment]
```

Comment:
We might want to add a command for updating just the map (-r FILENAME) just in case?

Reply:
This is a command for just the maps!  Or you mean like only one of the directories?

Reply:
This is just for the two map CSVs?

Reply:
I think Jess is saying this command _is_ only the map files. Correct?

Reply:
Yes, sorry, I ultimately had to try my comment several times that particular day because it wasn't working so it got briefer and briefer haha.  Yes, this command only does the map files.